src/app/components/accounts/activation-email/activation-email.component.ts

```diff
@@ -28,12 +28,11 @@ export class ActivationEmailComponent implements OnInit {
         });
 
         this.registerService.postActivation(this.uuid, this.token)
-            .subscribe(() => {
-                this.toastr.success('You have activated your account successfully.');
-                this.router.navigate(['/accounts/login']).then();
-            }, error => {
-                this.toastr.error(error);
-                this.router.navigate(['/accounts/login']).then();
+            .subscribe((result) => {
+                if (result.success) {
+                    this.toastr.success('You have activated your account successfully.');
```

Comment:
Message should go in the service.

Reply:
This will be handled in another PR as discussed. Same for other similar comments